
laser_stitcher_output:
  cloud_list:           [nav_cloud, local_dense_cloud]
  nav_cloud: 
    incremental_update: true
    pub_throttle:       1
    retain_after_scan:  true
    publish:            true
    save:               false
    # Postprocessing Data
    should_postprocess:             true
    min_cloud_size:                 50
    throttle_postprocess:           true
    postprocess_throttle_max:       50
    # NOTE - task names in task_list below MUST match the subheadings under the tasks heading
    task_list:            [ clip, voxelize, transform ]
    tasks:
      transform:
        type:             1     # Transformation
        map_name:         map
        should_publish:   true
        publish_topic:    transformed_cloud
      clip: 
        type:             3     # Clipping (Cond)
        box:              [ -5, 5, -3, 3, -1, 3.7 ]
        keep_ordered:     false
        should_publish:   true
        publish_topic:    clipped_cloud
      voxelize:           
        type:             4     # Voxelization
        leaf_sizes:       [ 0.03, 0.03, 0.03 ]
        keep_ordered:     false
        should_publish:   true
        publish_topic:    voxelized_cloud
  local_dense_cloud: 
    incremental_update: false
    pub_throttle:       1
    retain_after_scan:  false
    publish:            true
    save:               true
    # Postprocessing Data
    should_postprocess:             false